src/test/scala/com/gu/identity/play/CookieBuilderTest.scala

```diff
@@ -0,0 +1,69 @@
+package com.gu.identity.play
+
+import org.joda.time.format.ISODateTimeFormat
+import org.joda.time.{DateTime, Duration}
+import org.scalactic.Tolerance._
```

Comment:
I'm curious, this looks like the only use of scalactic, but where is it actually getting used? Do we actually need the dependency?